src/lib/types/stores/client-instance-store.ts

```diff
@@ -26,6 +26,9 @@ export interface IClientInstanceStore
         environment: string,
     ): Promise<IClientInstance[]>;
     getBySdkName(sdkName: string): Promise<IClientInstance[]>;
+    groupApplicationsBySdk(): Promise<
```

Comment:
following existing pattern. We can consider splitting into read models when we move to feature oriented architecture